sale_layout_category_hide_detail/__manifest__.py

```diff
@@ -15,11 +15,11 @@
         "sale_management",
     ],
     "data": [
-        "views/sale_layout_category_view.xml",
-        "views/sale_order_report_templates.xml",
-        "views/sale_portal_templates.xml",
```

Comment:
Why have you remove portal templates? Aren't they needed?

Reply:
The portal sale order template was removed wrongly, but regarding to the portal invoice, now in v12 there isn't portal invoice template, odoo use the same template of the PDF report. That template is rendered and shown in a <frame>. So, fortunately we don't have to redefine a portal invoice template :)